app/models/ability.rb

```diff
@@ -26,9 +26,15 @@ def initialize(user)
       can :manage, WebsiteMember do |website_member|
         user.is_owner_of?(website_member.website)
       end
-      can :read, WebsiteMember do |website_member|
+      can [:read, :update], WebsiteMember, [user_id: user.id] do |website_member|
```

Comment:
why are you keeping the `user.is_member_of?` below. That has the same usage as the `user_id: user.id` call.
This line should be: `can [:read, :update], WebsiteMember, user_id: user.id`